source/setup/pc_setup.rst

```diff
@@ -53,3 +53,38 @@ from the host machine to the VM. Follow these instructions in VirtualBox.
         #. Enable Network Adapter
         #. Attached to Bridged Adapter
         #. Name should be your wired adapter
+        
+******************
+Using Hyper-V instead of Virtual Box
+******************
+If you using windows OS and already running hyper-v system, virtual box will not be able to start the VM. 
+These instruction is converting the VM from virtual box to hyper-v.
+Converting directly via microsoft converted wont work, some VM description is not supported, hence need to extract the VM description info, edit and override it before convert to hyper-v. Steps are below.
```

Comment:
probably better to put a `.. warning::` up near where it says to install virtualbox, then link to the section down here (or a separate page)